tests/test_prtype.py

```diff
@@ -66,7 +66,7 @@ async def test_news_only():
 
 
 async def test_docs_no_news():
-    filenames = {"path/to/docs1.rst"}
+    filenames = {"path/to/docs1.rst", "other/path/to/docs2.md"}
```

Comment:
Wouldn't this test pass even without the changes in `prtype.py`?
The filenames include an `.rst` file, so the `docs` label should be applied regardless.
I would suggest to verify this, and possibly turn the list of filenames into a fixture.

Reply:
> Wouldn't this test pass even without the changes in prtype.py?

No.  In this case classify_by_filepaths() will quickly return an empty list, walking over filenames (else branch).